### DISCUSSION TITLE
Select certification method not hitting.

### DISCUSSION THREAD

**Discussion by mukeshmishra996** [cefsharp/CefSharp#4345] [General]
I am trying to implement custom popup to select certificate from list of certificate. But OnSelectClientCertificate is not hitting. Can you please help me on this.

**Comment by amaitland** [comment]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask support related  questions.

> But OnSelectClientCertificate is not hitting. Can you please help me on this.

There's an example contained within the source. Past that if you require assistance then you need to provide more detail, code example, version, winforms/wpf etc. See https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following
